### PR TITLE
configured code checkstyle validation

### PR DESCRIPTION
**Pull request by boroda4436**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -23,6 +25,27 @@
         </dependency>
     </dependencies>
     <build>
+        <plugins>
```
Comment (roman-mate) [inline pom.xml]: this chunk seems to be duplicated in two places. is that expected?

File: src/main/conf/checkstyle.xml [hunk 2]
```diff
@@ -0,0 +1,256 @@
+<?xml version="1.0"?>
```
Comment (roman-mate) [inline src/main/conf/checkstyle.xml]: it looks like we'll have to duplicate this config in every repo? Is there a way to avoid duplication?

Comment (boroda4436) [reply src/main/conf/checkstyle.xml]: @roman-mate we might create a template repo as I wrote in the slack. However, I think this config will be copied automatically each time we will create a new repo. Multiple copies of this config file is a disadvantage of the approach with a separate repo for each HW.

I am not sure that we might configure checkstyle check in another way

File: pom.xml (hunk 1 above)
Comment (boroda4436) [reply pom.xml]: Fixed

File: src/main/conf/checkstyle.xml (hunk 2 above)
Comment (boroda4436) [reply src/main/conf/checkstyle.xml]: @roman-mate  we may try to access this file from the web instead of using this locally

Comment (boroda4436) [reply src/main/conf/checkstyle.xml]: @roman-mate fixed

Comment (roman-mate) [reply src/main/conf/checkstyle.xml]: Can this file be removed now?

**Pull request merged**